MUMS.R

```diff
@@ -60,10 +62,6 @@ MUMS = function(donors,Start,End,core=T,prices="current",disb=T,channel=""){
 	dat = Map(namefunk, dat, 1:length(dat))
 	dat = as.data.frame(rbindlist(dat)) %>% pivot_wider(names_from=Var,values_from=value)
 	dat = remove_constant(dat)
-	names(dat)[names(dat)=="TIME_PERIOD"] = "year"
+	setnames(dat, "TIME_PERIOD", "year")
```

Comment:
ahh I didn't know this existed, I have loads of scripts where names(df)[names(df)=="X"]= "Y" take up like 10+ lines, and always thought there must be a better way. 
Just checking though is this a change for readability or performance?

Reply:
It's a bit of both. I don't know how it affects how long it takes to execute, but I like how verbose `setnames` is in terms of a function, and the function also first checks that the names you're trying to change exist and will throw an error if they don't.

You can also pass arrays to the first and second arguments, so you can rename a bunch of variables at once.